### DISCUSSION TITLE
Dropdown menu becomes unclickable

### DISCUSSION THREAD

**Discussion by anthonyfrancq** [PaperStrike/Pjax#140] [Q&A] [answered]
I've noticed that this will break the drop down menu's in HTML, they will not be clickable anymore (the drop down portion does nothing).

This occurs not on the initial load but any other link that is clicked after, however this does not happen every time I would say 50% of the time or more.

Please see [https://vimeo.com/597953276](url)

**Comment by PaperStrike** [comment, marked as answer]
Can you provide a demo site for me to check the source code? The video provides very limited information and I can only guess if #102 is the blocker, but I can't be sure. It is not possible for me to do any targeted fixes until the problem is identified, and you may need to check again after #102 has been implemented to see if the problem still exists.

**Reply by anthonyfrancq** [reply]
https://www.affwise.io/dashboard/login.php

User: test@test.com
password: test

**Reply by PaperStrike** [reply]
@anthonyfrancq The problem seems to be caused by your refresh section being too large ([`selectors`](https://github.com/PaperStrike/Pjax#selectors) selecting the entire `body`). Since Pjax always executes the scripts in the refresh section, all your scripts in `<body>` such as `tabler.min.js` and `pjax.js` are executed once after each switch. And at least those two scripts in your page are designed to be run only once. Running `tabler.min.js` twice causes the click event to be handled twice, causing the menu to **expand and collapse quickly** as if nothing had been done. Running `pjax.js` twice will cause an `Identifier 'pjax' has already been declared` error in the console.

Therefore, using more specific selectors should solve the dropdown menu problem.

---

FWIW, the `pjax.js`, `pjax.min.js`, ... that we provide should all be used as single files, don't edit them. Put your custom Pjax code into other JS file(s).

Another issue, the source map file (`pjax.js.map`) in your server is out of date.

**Reply by anthonyfrancq** [reply]
Thank you for the recommendations, solution has been resolved.
